backend/src/main/java/com/library/backend/service/MealService.java

```diff
@@ -22,30 +26,63 @@ public List<Meal> getAllMeals() {
     }
 
 
-    public Meal getMealById(String id) {
+    public Meal getMealById(String id) throws Exception {
+        Meal meal = mealRepository.findById(id).orElseThrow(() -> new Exception("ID not found."));
```

Comment:
maybe you can add a custom Exception like IdNotFoundException -> it would tell you what could happen at first glance